README.md

```diff
@@ -76,75 +79,190 @@ git -C web5-spec sparse-checkout set test-vectors
 
-To kick that off:
+These SNAPSHOTs are generated and published whenever there's a new push to `main`. If you want to manually kick that off to preview some changes introduced in a PR branch:
 
-1. Open the [Publish workflow](https://github.com/TBD54566975/web5-kt/actions/workflows/publish.yml), press the **Run workflow button** selecting the branch you want to generate the snapshot from.
+1. Open the [SDK Kotlin CI Workflow](https://github.com/TBD54566975/web5-kt/actions/workflows/ci.yml), press the **Run workflow button** selecting the branch you want to generate the snapshot from.
 
-2. In the version field, insert the current version, a short meaningful identifier and the `-SNAPSHOT` prefix, ie:
+2. In the version field, insert the current version, a short meaningful identifier and the `-SNAPSHOT` suffix, ie:
 
    - 0.11.0.pr123-SNAPSHOT
    - 0.11.0.shortsha-SNAPSHOT
    - 0.11.0.fixsomething-SNAPSHOT
 
 3. Run workflow!
 
-**DON'T FORGET THE `-SNAPSHOT` SUFFIX**, otherwise it will generate publish a new official release to maven registry.
+You **MUST** use the `-SNAPSHOT` suffix, otherwise it's not a valid preview `SNAPSHOT` and it will be rejected.
+
+`SNAPSHOT`s will be available in [TBD's Artifactory `tbd-oss-snapshots-maven2` Repository](https://blockxyz.jfrog.io/artifactory/tbd-oss-snapshots-maven2).
 
 ### Releasing New Versions
 
-To release a new version, just execute the following steps:
+To release a new version, execute the following steps:
 
-1. Open the [Publish workflow](https://github.com/TBD54566975/tbdex-kt/actions/workflows/publish.yaml), press the **Run workflow button** and leave the main branch selected (unless its a rare case where you don't want to build the main branch for the release).
+1. Open the [Release and Publish](https://github.com/TBD54566975/web5-kt/actions/workflows/release.yml), press the **Run workflow button** selecting the branch you want to generate the snapshot from.
 
-2. In the version field, insert the new version to be released, ie: 0.12.3-beta
+2. In the version field, declare the version to be released. ie:
 
-3. Run workflow! The package will be built and **published to maven central**, **docs will be published** (see below) and **the GitHub release will be automatically generated**!
+   - 0.15.2
+   - 0.17.0-alpha-3
+   - 1.6.3
 
-## Publishing Docs
+   - **Choose an appropriate version number based on semver rules. Remember that versions are immutable once published to Maven Central; they cannot be altered or removed.**
+
+3. Press the **Run workflow button** and leave the main branch selected (unless its a rare case where you don't want to build from the main branch for the release).
+
+4. Run workflow! This:
+
+- Builds
+- Tests
+- Creates artifacts for binaries and sources
+- Signs artifacts
+- Uploads artifacts to TBD Artifactory
+- Tags git with release number "v$version"
```

Comment:
does this mean we'll create git tags like `v0.11.0-somesha-SNAPSHOT` ?

Reply:
oh wait maybe not, this is under the release section

Reply:
Correct, git tags are only for releases